React/tic-tac/src/app/screens/Login/components/Form.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import validate from '../components/formField/validate';
+
+import style from './styles.scss';
+import renderField from './formField/index';
+
+const loginForm = props => {
```

Comment:
Apply the props destructuring in the function param
```suggestion
const loginForm = ({ handleSubmit, onSubmit, submitting, onClick }) => {
```